src/algebra/order/group.lean

```diff
@@ -1146,6 +1146,10 @@ abs_le.2 ⟨(neg_add (|a|) (|b|)).symm ▸
   add_le_add (neg_le.2 $ neg_le_abs_self _) (neg_le.2 $ neg_le_abs_self _),
   add_le_add (le_abs_self _) (le_abs_self _)⟩
 
+lemma abs_add' {α : Type*} [linear_ordered_add_comm_group α] (a b : α) :
```

Comment:
Can't you use the ambient `linear_ordered_add_comm_group` here?

Reply:
Oh thanks, I meant to make that change and forgot!